### ISSUE TITLE
Disable pinch and zoom on the splash screen

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/joist#672]
From #671, it is currently possible to pinch and zoom in on the splash screen while the sim is loading using native browser zoom. If this is done it may be impossible to zoom back out because once the sim is loaded we are preventing default pan/zoom gestures and commands.

So we would like to disable pinch to zoom on the splash screen. This will involve preventing default on certain gestures while the sim is loading. Splash.js looks like it accounts for scale well when the window is resized such as on ctrl +/- or other browser zoom. So I think trackpad and touch screen pinch to zoom are the only ones we have to prevent.

**Comment by jessegreenberg** [comment]
This is done with the above commit. I tested on an iPad, Windows 10 touch screen track pad, and macOS trackpad. @samreid would you mind reviewing this change since you are the author of splash.js?

**Comment by samreid** [comment]
I tested with and without the commit on iPad 7 and saw that it had the desired behavior, nice work!  I made minor formatting changes in the commit.  Some questions:

- [ ] It seems there is a typo or contradiction in this sentence `const passiveOption = { passive: false }; // mark as passive to allow preventDefault()`, we are marking it as passive:false?  I think the doc probably just needs to be updated since we do need `passive:false` to prevent default.
- [ ] What do you think about inlining the options instead of factoring them out?
- [ ] `gesturestart` is listed as nonstandard, see https://developer.mozilla.org/en-US/docs/Web/API/Element/gesturestart_event.  Is this what gets the solution to work on Safari and Mobile Safari?  Perhaps document that it is known to be nonstandard, but important to handle anyways.
- [ ] How can we be certain that 'wheel', 'touchstart', 'gesturestart' is the full list we need?  I saw your point about keyboard not being necessary.

**Comment by jessegreenberg** [comment]
Thanks for reviewing! For the first two items, I changed as you recommended. I added documentation about `gesturestart`. It is required for macOS trackpad gestures at least, but I think `touchstart` covers iOS touchscreens. I added documentation to this effect.

> How can we be certain that 'wheel', 'touchstart', 'gesturestart' is the full list we need?

Hmm, I am not sure. These three cover mouse, trackpad, and touch events which are the native methods of input I can think of to request zoom. I reviewed https://developer.mozilla.org/en-US/docs/Web/Events and didn't see any others that should be added. [msContentZoom](https://developer.mozilla.org/en-US/docs/Web/Events/msContentZoom) caught my eye, but it is specific to IE11 so I don't think it should be. It used to be for edge too but not anymore since the Chromium switch.
